### PR TITLE
New WFP sensor cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR contains new sensor cal csvs for the following WFPs:
Coastal - 12936-01, 13103-02, 13103-03, 13103-04, 13103-05
Global - 12936-02, 13104-01, 13104-02

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-instrument-cal-csvs-dt-69)

File: calibration/DOFSTK/CGINS-DOFSTK-02397__20191203.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2397,CC_frequency_offset,-8.192696e+002, source_file DOFST-K_SBE43F_SN_2397_Calibration_2019-12-03.zip > 2397.cal and SBE 43F O2397 SOCAdjusted 16Dec19.pdf
+43-2397,CC_oxygen_signal_slope,2.5449e-004, adjusted Soc value
+43-2397,CC_residual_temperature_correction_factor_a,-3.866776e-003,
+43-2397,CC_residual_temperature_correction_factor_b,2.181303e-004,
+43-2397,CC_residual_temperature_correction_factor_c,-2.817617e-006,
+43-2397,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (dbogorff) [inline calibration/DOFSTK/CGINS-DOFSTK-02397__20191203.csv]: Looks Good!

File: calibration/DOFSTK/CGINS-DOFSTK-02628__20191203.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2628,CC_frequency_offset,-8.047682e+002, source_file DOFST-K_SBE43F_SN_2628_Calibration_2019-12-03.zip > 2628.cal and SBE 43F O2628 SOCAdjusted 16Dec19.pdf
+43-2628,CC_oxygen_signal_slope,3.1886e-004, adjusted Soc value
+43-2628,CC_residual_temperature_correction_factor_a,-3.619693e-003,
+43-2628,CC_residual_temperature_correction_factor_b,1.813512e-004,
+43-2628,CC_residual_temperature_correction_factor_c,-3.070325e-006,
+43-2628,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (dbogorff) [inline calibration/DOFSTK/CGINS-DOFSTK-02628__20191203.csv]: Looks Good!  (DB, JK)

File: calibration/DOFSTK/CGINS-DOFSTK-02724__20191204.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2724,CC_frequency_offset,-7.751549e+002, source_file DOFST-K_SBE43F_SN_2724_Calibration_2019-12-04.zip > 2724.cal and SBE 43F O2724 SOCAdjusted 18Dec19.pdf
+43-2724,CC_oxygen_signal_slope,2.9919e-004,adjusted Soc value
+43-2724,CC_residual_temperature_correction_factor_a,-3.576287e-003,
+43-2724,CC_residual_temperature_correction_factor_b,1.610209e-004,
+43-2724,CC_residual_temperature_correction_factor_c,-1.945635e-006,
+43-2724,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (dbogorff) [inline calibration/DOFSTK/CGINS-DOFSTK-02724__20191204.csv]: Looks good!  (DB, JK)

File: calibration/DOFSTK/CGINS-DOFSTK-02725__20191204.csv [hunk 4]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2725,CC_frequency_offset,-7.782622e+002, source_file DOFST-K_SBE43F_SN_2725_Calibration_2019-12-04.zip > 2725.cal and SBE 43F O2725 SOCAdjusted 16Dec19.pdf
+43-2725,CC_oxygen_signal_slope,2.9868e-004,adjusted Soc value
+43-2725,CC_residual_temperature_correction_factor_a,-2.907732e-003,
+43-2725,CC_residual_temperature_correction_factor_b,1.135953e-004,
+43-2725,CC_residual_temperature_correction_factor_c,-1.255577e-006,
+43-2725,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (dbogorff) [inline calibration/DOFSTK/CGINS-DOFSTK-02725__20191204.csv]: Looks Good!  (DB, JK)

File: calibration/DOFSTK/CGINS-DOFSTK-02726__20191211.csv [hunk 5]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2726,CC_frequency_offset,-8.085641e+002, source_file DOFST-K_SBE43F_SN_2726_Calibration_2019-12-11.zip > 2726.cal and SBE 43F O2726 SOCAdjusted 18Dec19.pdf
+43-2726,CC_oxygen_signal_slope,2.5626e-004, adjusted Soc value
+43-2726,CC_residual_temperature_correction_factor_a,-2.838964e-003,
+43-2726,CC_residual_temperature_correction_factor_b,1.492261e-004,
+43-2726,CC_residual_temperature_correction_factor_c,-1.822826e-006,
+43-2726,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (dbogorff) [inline calibration/DOFSTK/CGINS-DOFSTK-02726__20191211.csv]: Looks good!  (DB, JK)

File: calibration/FLORDL/CGINS-FLORDL-02736__20191206.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+2736,CC_angular_resolution,1.096,Constant; source_file FLORD-L_FLBBRTD_SN_2736_Calibration_2019-12-06.zip > FLBBRTD-2736 Char Sheets.pdf
+2736,CC_dark_counts_chlorophyll_a,50,
+2736,CC_dark_counts_volume_scatter,48,
+2736,CC_depolarization_ratio,0.039,Constant
+2736,CC_measurement_wavelength,700,Constant
+2736,CC_scale_factor_chlorophyll_a,0.0071,
+2736,CC_scale_factor_volume_scatter,1.808E-06,
+2736,CC_scattering_angle,140,Constant
```
Comment (dbogorff) [inline calibration/FLORDL/CGINS-FLORDL-02736__20191206.csv]: Looks good!  (DB, JK)

File: calibration/FLORDL/CGINS-FLORDL-03280__20191206.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3280,CC_angular_resolution,1.096,Constant; source_file FLORD-L_FLBBRTD_SN_3280_Calibration_2019-12-06.zip > FLBBRTD-3280 Charsheets.pdf
+3280,CC_dark_counts_chlorophyll_a,47,
+3280,CC_dark_counts_volume_scatter,46,
+3280,CC_depolarization_ratio,0.039,Constant
+3280,CC_measurement_wavelength,700.0,Constant
+3280,CC_scale_factor_chlorophyll_a,0.0066,
+3280,CC_scale_factor_volume_scatter,1.778e-06,
+3280,CC_scattering_angle,140.0,Constant
```
Comment (dbogorff) [inline calibration/FLORDL/CGINS-FLORDL-03280__20191206.csv]: Looks good!  (DB, JK)

File: calibration/FLORDL/CGINS-FLORDL-03283__20191206.csv [hunk 8]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3283,CC_angular_resolution,1.096,Constant; source_file FLORD-L_FLBBRTD_SN_3283_ Calibration_2019-12-06.zip > FLBBRTD-3283_CharSheets.pdf
+3283,CC_dark_counts_chlorophyll_a,46,
+3283,CC_dark_counts_volume_scatter,46,
+3283,CC_depolarization_ratio,0.039,Constant
+3283,CC_measurement_wavelength,700,Constant
+3283,CC_scale_factor_chlorophyll_a,0.0067,
+3283,CC_scale_factor_volume_scatter,1.778E-06,
+3283,CC_scattering_angle,140,Constant
```
Comment (dbogorff) [inline calibration/FLORDL/CGINS-FLORDL-03283__20191206.csv]: Looks good!  (DB, JK)

File: calibration/FLORTK/CGINS-FLORTK-00969__20191210.csv [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+969,CC_angular_resolution,1.076, source_file FLORT-K_BBFL2_SN_969_Calibration_2019-12-10.zip > BBFL2-969 Charsheets.pdf
+969,CC_dark_counts_cdom,51,
+969,CC_dark_counts_chlorophyll_a,50,
+969,CC_dark_counts_volume_scatter,49,
+969,CC_depolarization_ratio,0.039,
+969,CC_measurement_wavelength,700,
+969,CC_scale_factor_cdom,0.0487,
+969,CC_scale_factor_chlorophyll_a,0.0115,
+969,CC_scale_factor_volume_scatter,3.690E-06,
+969,CC_scattering_angle,124,
```
Comment (dbogorff) [inline calibration/FLORTK/CGINS-FLORTK-00969__20191210.csv]: Looks good!  (DB, JK)

File: calibration/FLORTK/CGINS-FLORTK-01100__20191210.csv [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+1100,CC_angular_resolution,1.076, source_file FLORT-K_BBFL2_SN_1100_Calibration_2019-12-10.zip > BBFL2-1100 Charsheets.pdf
+1100,CC_dark_counts_cdom,49,
+1100,CC_dark_counts_chlorophyll_a,49,
+1100,CC_dark_counts_volume_scatter,49,
+1100,CC_depolarization_ratio,0.039,
+1100,CC_measurement_wavelength,700,
+1100,CC_scale_factor_cdom,0.0506,
+1100,CC_scale_factor_chlorophyll_a,0.0113,
+1100,CC_scale_factor_volume_scatter,3.476E-06,
+1100,CC_scattering_angle,124,
```
Comment (dbogorff) [inline calibration/FLORTK/CGINS-FLORTK-01100__20191210.csv]: Looks good!  (DB, JK)

File: calibration/FLORTK/CGINS-FLORTK-01101__20191210.csv [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+1101,CC_angular_resolution,1.076, source_file FLORT-K_BBFL2_SN_1101_Calibration_2019-12-10.zip > BBFL2-1101 Charsheets.pdf
+1101,CC_dark_counts_cdom,50,
+1101,CC_dark_counts_chlorophyll_a,50,
+1101,CC_dark_counts_volume_scatter,49,
+1101,CC_depolarization_ratio,0.039,
+1101,CC_measurement_wavelength,700,
+1101,CC_scale_factor_cdom,0.0553,
+1101,CC_scale_factor_chlorophyll_a,0.0114,
+1101,CC_scale_factor_volume_scatter,3.476E-06,
+1101,CC_scattering_angle,124,
```
Comment (dbogorff) [inline calibration/FLORTK/CGINS-FLORTK-01101__20191210.csv]: Looks good!  (DB, JK)

File: calibration/FLORTK/CGINS-FLORTK-01118__20191210.csv [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+1118,CC_angular_resolution,1.076, source_file FLORT-K_BBFL2_SN_1118_Calibration_2019-12-10.zip > BBFL2-1118 Charsheets.pdf
+1118,CC_dark_counts_cdom,48,
+1118,CC_dark_counts_chlorophyll_a,53,
+1118,CC_dark_counts_volume_scatter,49,
+1118,CC_depolarization_ratio,0.039,
+1118,CC_measurement_wavelength,700,
+1118,CC_scale_factor_cdom,0.0527,
+1118,CC_scale_factor_chlorophyll_a,0.0106,
+1118,CC_scale_factor_volume_scatter,3.341E-06,
+1118,CC_scattering_angle,124,
```
Comment (dbogorff) [inline calibration/FLORTK/CGINS-FLORTK-01118__20191210.csv]: Looks good!  (DB, JK)

File: calibration/FLORTK/CGINS-FLORTK-01119__20191210.csv [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+1119,CC_angular_resolution,1.076, source_file FLORT-K_BBFL2_SN_1119_Calibration_2019-12-10.zip > BBFL2-1119 Charsheets.pdf
+1119,CC_dark_counts_cdom,51,
+1119,CC_dark_counts_chlorophyll_a,51,
+1119,CC_dark_counts_volume_scatter,48,
+1119,CC_depolarization_ratio,0.039,
+1119,CC_measurement_wavelength,700,
+1119,CC_scale_factor_cdom,0.0550,
+1119,CC_scale_factor_chlorophyll_a,0.0111,
+1119,CC_scale_factor_volume_scatter,3.471E-06,
+1119,CC_scattering_angle,124,
```
Comment (dbogorff) [inline calibration/FLORTK/CGINS-FLORTK-01119__20191210.csv]: Looks good!  (DB, JK)

File: calibration/PARADK/CGINS-PARADK-20423__20191206.csv [hunk 14]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20423,CC_dark_offset,1.2, source_file PARAD-K_QSP2200_SN_20423_Calibration_2019-12-06.pdf
+20423,CC_scale_wet,9.58E-18,
```
Comment (dbogorff) [inline calibration/PARADK/CGINS-PARADK-20423__20191206.csv]: Looks good!  (DB, JK)

File: calibration/PARADK/CGINS-PARADK-20465__20191206.csv [hunk 15]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20465,CC_dark_offset,1.3, source_file PARAD-K_QSP2200_SN_20465_Calibration_2019-12-06.pdf
+20465,CC_scale_wet,9.39E-18,
```
Comment (dbogorff) [inline calibration/PARADK/CGINS-PARADK-20465__20191206.csv]: Looks good!  (DB, JK)

File: calibration/PARADK/CGINS-PARADK-20466__20191206.csv [hunk 16]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20466,CC_dark_offset,1.2, source_file PARAD-K_QSP2200_SN_20466_Calibration_2019-12-06.pdf
+20466,CC_scale_wet,8.95E-18,
```
Comment (dbogorff) [inline calibration/PARADK/CGINS-PARADK-20466__20191206.csv]: Looks good!  (DB, JK)

File: calibration/PARADK/CGINS-PARADK-20468__20191206.csv [hunk 17]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20468,CC_dark_offset,1.0, source_file PARAD-K_QSP2200_SN_20468_Calibration_2019-12-06.pdf
+20468,CC_scale_wet,9.67E-18,
```
Comment (dbogorff) [inline calibration/PARADK/CGINS-PARADK-20468__20191206.csv]: Looks good!  (DB, JK)

File: calibration/PARADK/CGINS-PARADK-20582__20191206.csv [hunk 18]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20582,CC_dark_offset,1.2, source_file PARAD-K_QSP2200_SN_20582_Calibration_2019-12-06.pdf
+20582,CC_scale_wet,1.02E-17,
```
Comment (dbogorff) [inline calibration/PARADK/CGINS-PARADK-20582__20191206.csv]: Looks good!  (DB, JK)

**Pull request merged**